tests/system/action/motion/test_update_metadata.py

```diff
@@ -38,6 +38,45 @@ def test_update_correct(self) -> None:
         assert model.get("tag_ids") == []
         assert model.get("attachment_ids") == []
 
+    def test_update_metadata_workflow_id(self) -> None:
+        self.create_model("meeting/2538", {"name": "name_jkPIYjFz"})
+        self.create_model(
+            "motion/111", {"name": "name_srtgb123", "meeting_id": 2538, "state_id": 88}
```

Comment:
add recommendation_id here to make sure it is reset

Reply:
Done.